toolium/driver_wrapper.py

```diff
@@ -264,26 +266,30 @@ def connect_playwright(self):
         """
         async_loop = self.async_loop
         self.playwright = async_loop.run_until_complete(async_playwright().start())
-        # TODO: select browser from config
-        headless_mode = self.config.getboolean_optional('Driver', 'headless')
-        self.playwright_browser = async_loop.run_until_complete(self.playwright.chromium.launch(headless=headless_mode))
-        self.driver = async_loop.run_until_complete(self.playwright_browser.new_page())
+
+        # In case of using a persistent context this property must be set and a BrowserContext is returned instead of a Browser
+        user_data_dir = self.config.get_optional('PlaywrightContextOptions', 'user_data_dir', None)
+        if user_data_dir:
```

Comment:
to avoid duplications, config driver could be initialized before if/else
`config_driver = ConfigDriver(self.config, self.utils, self.playwright)`